distribution/docs/src/main/resources/_contents/_security-contents/managing-security-contents.adoc

```diff
@@ -27,68 +27,639 @@ From the Web Admin Console, the following configurations are available from a st
 |Configuration ID
 |Description
 
-|Login Page
-|`org.codice.${ddf-branding-lowercase}security.handler.guest.configuration`
+|<<login_page_table,Login Page>>
+|`org.codice.${ddf-branding-lowercase}.security.handler.guest.configuration`
```

Comment:
It probably is. I started it because I'd get a comment on it on every PR when I wasn't doing it. I have no problem dropping it if people don't like it.